added for loops to let the api try again after error timeout

tal-sitton opened the pull request.

tal-sitton committed:
added for loops to let the api try again after error timeout

TomerSitton commented on AlbumDownloader.py:
```diff
@@ -1,4 +1,5 @@
 import re
+import socket
```
Import only the things u want from socket

TomerSitton commented on AlbumDownloader.py:
```diff
@@ -301,10 +304,32 @@ def download_song(song_title, artist, output_dir, wanted_length=None):
-    respond = request.execute()
+
+    for i in range(3):
+        z = 0
```
What the fuck is z

TomerSitton commented on AlbumDownloader.py:
```diff
@@ -427,7 +452,7 @@ def main():
                 continue
-            add_mp3_metadata(file_path=song_path, title=song_title, album=album_title, artist=artist, index=i + 1)
+            # add_mp3_metadata(file_path=song_path, title=song_title, album=album_title, artist=artist, index=i + 1)
 
```
Undo #

tal-sitton committed:
imported only timeout from socket
changed the var z to a real thing, just nade that the "respond" and "videos" will be defined as None if the youtube api failed.
